integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -166,11 +166,11 @@ resources:
         {{- if contains "," .mediumtype }}
           {{- $type := .type }}
           {{- $path := .path }}
-          {{- $split := split "," .mediumtype }}
-          {{- range $key, $val := $split }}
+          {{- $parts := splitList "," .mediumtype }}
+          {{- range $i, $val := $parts }}
```

Comment:
`split "SSD,HDD"` gives a dict format `{_0: SSD, _1: HDD}`, whereas `splitList "SSD,HDD"` gives `[SSD, HDD]`, which is more intuitive and avoids the underscore replacing in the key.